test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala

```diff
@@ -0,0 +1,129 @@
+package com.typesafe.spark.test.mesos.framework.runners
+
+import java.io.File
+import java.net.URL
+
+import Utils._
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.spark.test.mesos.mesosstate.MesosCluster
+
+import org.apache.zookeeper.{ZKUtil, ZooKeeper, KeeperException}
+
+import scala.sys.process.Process
+
+object MultiClusterModeRunner {
```

Comment:
I prefer not to mix the existing one since this one differs from it with zk cleanup, and down the road I think we like to start adding more specific setup for multi dispatchers when it comes to multi-tenant setup.